packages/engine-server/src/markdown/remark/dendronPreview.ts

```diff
@@ -0,0 +1,51 @@
+import { APIUtils, AssetGetRequest } from "@dendronhq/common-all";
+import { vault2Path } from "@dendronhq/common-server";
+import _ from "lodash";
+import { Image } from "mdast";
+import path from "path";
+import Unified, { Transformer } from "unified";
+import { Node } from "unist";
+import visit from "unist-util-visit";
+import { VFile } from "vfile";
+import { EngineUtils } from "../../utils";
+import { DendronASTTypes } from "../types";
+import { MDUtilsV5 } from "../utilsv5";
+import { NoteRefsOpts } from "./noteRefs";
+
+type PluginOpts = NoteRefsOpts & {};
+
+function handleImage(proc: Unified.Processor, node: Image) {
```

Comment:
Not related to this code here, but do we have metrics implemented around the time it takes to finish rendering a preview?

Reply:
nope, but will add that as a next task